### ISSUE TITLE
:LspManageServers doesn't show all LSP servers.

### ISSUE THREAD

**Issue by yutaroishiwata** [mattn/vim-lsp-settings#664]
![Screenshot 2023-03-27 at 12 11 43](https://user-images.githubusercontent.com/28632796/227831839-528f6a5d-80aa-48c8-9bc1-5c61acd21f1b.png)

**Comment by mattn** [comment]
What do you mean "all" ? The entries in settings.json ?

**Comment by yutaroishiwata** [comment]
My understanding is that the `:LspManageServers` command shows all supported LSP servers, is that wrong?

**Comment by mattn** [comment]
Yes. However, some LSP servers require commands for installation. For example, typescript-language-server require npm command. Probably, the missing servers are not shown since you dont have the command.

**Comment by yutaroishiwata** [comment]
The npm command is already installed. By the way, I am using Mac vim, and vim started from the terminal showed the required LSP server. I don't think this is a plugin issue, so you can close this issue. If you have any ideas as to the cause, it would be greatly appreciated if you could let us know.

`:version` Mac vim
![Screenshot 2023-03-27 at 15 50 54](https://user-images.githubusercontent.com/28632796/227863263-038f756a-d7e0-4b79-960d-1b412924b522.png)
`:version` Vim from terminal
![Screenshot 2023-03-27 at 15 51 11](https://user-images.githubusercontent.com/28632796/227863269-6e8de831-2aba-4262-9f4c-2b032dffa480.png)
